.vs/VSWorkspaceState.json

```diff
@@ -0,0 +1,6 @@
+{
+  "ExpandedNodes": [
+    ""
+  ],
+  "PreviewInSolutionExplorer": false
+}
```

Comment:
Same as `.vs/ProjectSettings.json` file. Probably better to be not included in the remote repository (github repo).